synapse/api/ratelimiting.py

```diff
@@ -186,6 +212,8 @@ def ratelimit(
         """Checks if an action can be performed. If not, raises a LimitExceededError
 
         Args:
+            requester: The requester that is doing the action, if any. Used to check for
+                ratelimit overrides.
```

Comment:
ditto.

I think we should also say something about this in the docstring for `can_requester_do_action`.